apps/server/src/modules/h5p-editor/h5p-editor.module.ts

```diff
@@ -57,7 +60,36 @@ const imports = [
 
 const controllers = [H5PEditorController];
 
-const providers = [Logger, H5PEditorUc, H5PEditorService, H5PPlayerService, H5PAjaxEndpointService, ...storages];
+const providers = [
+	{
+		provide: 'S3_Client',
+		scope: Scope.REQUEST,
```

Comment:
Why did you set it to Request scope? (I'm not saying it's wrong. Just asking out of interest.)

Reply:
There were some problems where S3 had unexplained behavior while working on the FWU project,  and creating the connection every request solved the problem